.travis.yml

```diff
@@ -79,10 +79,10 @@ matrix:
     deploy:
       provider: pages
       skip_cleanup: true
-      github_token: $github_token
+      github_token: $GITHUB_TOKEN
       local_dir: docs/_site
       on:
-        branch: master
+        branch: gh-pages
```

Comment:
This will only trigger if the PR is open on gh-pages (i believe), the `target_branch` is responsible for pushing `local_dir` to the `gh-pages` branch. 

I may be wrong, but this is what I recall